arrays-intermediate.js

```diff
@@ -17,10 +21,14 @@ function sumArray(numbers) {
 **************************************************************/
 function findFirstStringStartingWithLetter(letter, strings) {
   //TODO: Add your code here
+  strings = strings.map((value, index, array) => {
+    return value.toLowerCase();
+  });
+  console.log(strings.filter((value) => value.startsWith(letter)));
```

Comment:
You're supposed to find only one item using the `find` method, hence the name of the function `findFirst...`